tasks/system_probe.py

```diff
@@ -794,9 +794,6 @@ def kitchen_prepare(ctx, windows=is_windows, kernel_release=None, ci=False, pack
             if os.path.isdir(extra_path):
                 shutil.copytree(extra_path, os.path.join(target_path, extra))
 
-        if pkg.endswith("java"):
-            shutil.copy(os.path.join(pkg, "agent-usm.jar"), os.path.join(target_path, "agent-usm.jar"))
-
```

Comment:
AFAIK, the change in the file will make the CI to fail
It will not impact our docker size